lib/services/registered_addons.pm

```diff
@@ -78,10 +79,21 @@ sub check_suseconnect {
     diag "@addons";
 }
 
+sub check_suseconnect_cmd {
+    my $ls_out     = script_output("SUSEConnect --list-extensions", 120);
+    my $status_out = script_output("SUSEConnect --status-text",     120);
+    for (my $i = 1; $i < @addons; $i = $i + 1) {
+        diag "$addons[$i]";
+        die "$addons[$i] is not existed at SUSEConnect --list-extensions" if ($ls_out     !~ /Deactivate(.*)$addons[$i]/);
```

Comment:
I think here should check the register status for each module while not just only the module exist or not.

Reply:
I have check the register status. For SUSEConnect --list-extensions, register module has 'Deactivate' like:     Basesystem Module 15 SP2 ppc64le (Activated)
    Deactivate with: SUSEConnect -d -p sle-module-basesystem/15.2/ppc64le
For SUSEConnect --status-text, it only show registered addons.

Reply:
For SUSEConnect --status-text, we have checked registration status at system_prepre module